src/extensionConsole/pages/settings/ExperimentalSettings.tsx

```diff
@@ -63,7 +63,7 @@ const ExperimentalSettings: React.FunctionComponent = () => {
             isEnabled={performanceTracing}
             flag="performanceTracing"
           />
-          {isMV3() && (
+          {!isMV3() && (
```

Comment:
I find the ternary more readable in this case
```suggestion
          {isMV3() ? null : (
```